vita3k/app/src/app_init.cpp

```diff
@@ -118,44 +118,40 @@ void init_paths(Root &root_paths) {
 #if defined(__linux__) && !defined(__ANDROID__) && !defined(__APPLE__)
     // XDG Data Dirs.
     auto env_home = getenv("HOME");
-    auto env_data_dirs = getenv("XDG_DATA_DIRS");
-    auto env_data_home = getenv("XDG_DATA_HOME");
-    auto env_cache_home = getenv("XDG_CACHE_HOME");
-    auto env_config_home = getenv("XDG_CONFIG_HOME");
+    auto XDG_DATA_DIRS = getenv("XDG_DATA_DIRS");
+    auto XDG_DATA_HOME = getenv("XDG_DATA_HOME");
+    auto XDG_CACHE_HOME = getenv("XDG_CACHE_HOME");
+    auto XDG_CONFIG_HOME = getenv("XDG_CONFIG_HOME");
 
-    if (env_data_home != NULL)
-        root_paths.set_pref_path(fs::path(env_data_home) / app_name / app_name / dir_sep);
+    if (XDG_DATA_HOME != NULL)
+        root_paths.set_pref_path(fs::path(XDG_DATA_HOME) / app_name / app_name / dir_sep);
 
-    if (env_config_home != NULL)
-        root_paths.set_config_path(fs::path(env_config_home) / app_name / dir_sep);
+    if (XDG_CONFIG_HOME != NULL)
+        root_paths.set_config_path(fs::path(XDG_CONFIG_HOME) / app_name / dir_sep);
     else if (env_home != NULL)
         root_paths.set_config_path(fs::path(env_home) / ".config" / app_name / dir_sep);
 
-    if (env_cache_home != NULL) {
-        root_paths.set_cache_path(fs::path(env_cache_home) / app_name / dir_sep);
-        root_paths.set_log_path(fs::path(env_cache_home) / app_name / dir_sep);
+    if (XDG_CACHE_HOME != NULL) {
+        root_paths.set_cache_path(fs::path(XDG_CACHE_HOME) / app_name / dir_sep);
+        root_paths.set_log_path(fs::path(XDG_CACHE_HOME) / app_name / dir_sep);
     } else if (env_home != NULL) {
         root_paths.set_cache_path(fs::path(env_home) / ".cache" / app_name / dir_sep);
         root_paths.set_log_path(fs::path(env_home) / ".cache" / app_name / dir_sep);
     }
 
-    // Don't assume that base_path is portable.
-    if (fs::exists(root_paths.get_base_path() / "data") && fs::exists(root_paths.get_base_path() / "lang") && fs::exists(root_paths.get_base_path() / "shaders-builtin"))
-        root_paths.set_shared_path(root_paths.get_base_path());
-    else if (env_home != NULL)
+    if (env_home != NULL)
         root_paths.set_shared_path(fs::path(env_home) / ".local/share" / app_name / dir_sep);
 
-    if (env_data_dirs != NULL) {
-        auto env_paths = string_utils::split_string(env_data_dirs, ':');
+    if (XDG_DATA_DIRS != NULL) {
+        auto env_paths = string_utils::split_string(XDG_DATA_DIRS, ':');
         for (auto &i : env_paths) {
             if (fs::exists(fs::path(i) / app_name)) {
                 root_paths.set_shared_path(fs::path(i) / app_name / dir_sep);
                 break;
             }
         }
-    } else if (env_data_home != NULL) {
-        if (fs::exists(fs::path(env_data_home) / app_name / "data") && fs::exists(fs::path(env_data_home) / app_name / "lang") && fs::exists(fs::path(env_data_home) / app_name / "shaders-builtin"))
```

Comment:
Same here. If this is removed, and if XDG_DATA_DIRS is not defined, Vita3K will unconditionally override a detected data folder in base_path with the folder in .local/share. (Which may not be valid. Or may be missing required assets.) Breaking the standard release archive builds.